draft-huitema-dnssd-privacyscaling.xml

```diff
@@ -194,7 +210,7 @@ discovery secret.
 </t>
 </section>
 
-<section title="Discovery public key" anchor="discopubkey" >
+<section title="Shared public key" anchor="discopubkey" >
 <t>
```

Comment:
I would keep the term "discovery" rather than shared. Shared is ambiguous, because "shared key" typically implies that the member of the groups share a secret. I think we are missing something in the introduction, maybe a discussion of privacy properties of public keys.

Reply:
In this variant, the public key is a secret, is it not? That’s why I renamed it to “Shared” instead of “Discovery.” It is of course possible that I misread.

Reply:
Yes, the public key has to remain secret. Otherwise, adversaries could see the advertisements from the servers. They could try decrypting with known public key. If the decrypted value makes sense, they have identified the server.

Reply:
OTOH, take the "public printer" scenario. If the server is public, then it doesn't matter if its public key is disclosed.